tools/docker/builder/Dockerfile

```diff
@@ -24,6 +26,7 @@ RUN set -xe; \
         ERLANG_SHA256="89c98e177f70593a9f64f3c9962393a51e6730d16071820600d19d4aaa732412" && ERLANG_OS="amd64"; \
         JDK_SHA256="81f5bed21077f9fbb04909b50391620c78b9a3c376593c0992934719c0de6b73" && JDK_OS="x64"; \
         JQ_SHA256="af986793a515d500ab2d35f8d2aecd656e764504b789b66d7e1a0b727a124c44" && JQ_OS="linux64"; \
+        MOLD_SHA256="3893f89e5e0dcddcecc9f2ee17f14ad94fbf8b324eca45974b965353a50dac37" && MOLD_OS="x86_64-linux"; \
```

Comment:
downloaded and validated https://github.com/rui314/mold/releases/download/v1.3.1/mold-1.3.1-x86_64-linux.tar.gz

```
sha256sum mold-1.3.1-x86_64-linux.tar.gz
3893f89e5e0dcddcecc9f2ee17f14ad94fbf8b324eca45974b965353a50dac37  mold-1.3.1-x86_64-linux.tar.gz